airflow/www/static/js/dags.js

```diff
@@ -485,6 +515,8 @@ $(window).on('load', () => {
   $('body').on('mouseout', '.has-svg-tooltip', () => {
     hideSvgTooltip();
   });
+
+  refreshDagStats();
```

Comment:
`refreshDagStats` uses jQuery, so I brought it inside the page load callback with the other init functions.